doc/changelog/10-standard-library/12094-app_inj_tail.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:**
+  Extend app_inj_tail and other list lemmas to both directions
```

Comment:
Let us please have some formatting here:
```suggestion
  Extend ``app_inj_tail`` and other list lemmas to both directions
```

Reply:
Also, let's please list all the new lemmas, this happens to be the preferred style for the changelog.

Reply:
Done!